package.json

```diff
@@ -37,6 +36,9 @@
     "ts-node": "^10.9.2",
     "typescript": "^5.5.2"
   },
+  "peerDependencies": {
```

Comment:
NIT: Maybe move this right below the normal dependencies, since it's more important than the devDependencies.